package.json

```diff
@@ -57,6 +57,28 @@
       "description": "New tabs are opened at the top",
       "type": "bool",
       "value": false
+    },
+    {
+      "name": "largetabs",
+      "hidden": true,
+      "title": "Large Tabs",
+      "description": "Tabs are displayed large with thumbnail",
```

Comment:
That's only for the "Auto" setting. I also included an "Always" setting for having the tabs always display thumbnails.